Jenkinsfile

```diff
@@ -2,12 +2,14 @@
 
 @Library('pipeline-library') _
 
-def img
+def image
+def ws_only_image
```

Comment:
New variable definition for `ws_only_image`. Used to store a dedicated image for the `h` `websocket`